src/license.rs

```diff
@@ -46,8 +53,16 @@ pub fn file(crate_root: &Path, rel_dir: &Path, license_name: &str, single_licens
     let lic_abs_path = crate_root.join(lic_path);
     let spec_abs_path = crate_root.join(spec_path);
     let simple_abs_path = crate_root.join(simple_path);
+    let custom_abs_path = crate_root.join(license_path.unwrap_or(Path::new("")));
```

Comment:
How can this be made less ugly?

Reply:
depends on your taste. alternative
```suggestion
    let custom_abs_path = if let Some(lp) = license_path {
        crate_root.join(lp)
    } else {
        crate_root
    };
```

Reply:
So you are saying the above is fine. I do prefer a oneliner...

Reply:
oneliner is fine :+1: